OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -400,8 +400,10 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 				// Move production description from Tooltip to Buildable
 				if (engineVersion < 20161016)
 				{
-					var tooltipChild = node.Value.Nodes.FirstOrDefault(n => n.Key == "Tooltip");
-					if (tooltipChild != null || (tooltipChild = node.Value.Nodes.FirstOrDefault(n => n.Key == "DisguiseToolTip")) != null)
+					var tooltipChild = node.Value.Nodes.FirstOrDefault(n => n.Key == "Tooltip")
```

Comment:
Wouldn't it be simpler to use `var tooltipChild = node.Value.Nodes.FirstOrDefault(n => n.Key == "Tooltip" || n.Key == "DisguiseToolTip")`?